Foundation/NSXMLParser.swift

```diff
@@ -233,9 +233,9 @@ internal func _NSXMLParserStartElementNs(ctx: _CFXMLInterface, localname: Unsafe
     let parser = ctx.parser
     let reportQNameURI = parser.shouldProcessNamespaces
     let reportNamespaces = parser.shouldReportNamespacePrefixes
-    let prefixLen = prefix == nil ? strlen(UnsafePointer<Int8>(prefix)) : 0
+    let prefixLen = prefix == nil ? UInt(strlen(UnsafePointer<Int8>(prefix))) : 0
```

Comment:
Although it's true that size_t (and swift_size_t) is distinct from swift_ssize_t it still changes size based upon the underlying architecture.  It is ultimately typedef'd from __SIZE_TYPE__ which, although I couldn't find its actual definition) is 8 bytes on x86_64 (linux or mac) and 4 bytes on 32-bit arm.

I still don't think this is the best solution in this particular instance.  The error we're addressing here is:

error: cannot convert value of type 'UInt32' to expected argument type 'UInt'

So, it is my expectation that we're promoting a 32-bit unsigned into to a 64-bit unsigned int, or simply removing the bit-size specialization, with it remaining 32-bits.  Either way, it's safe in the 32-bit case.  In 64-bit code, this should be a no-op, because it's a cast from UInt to UInt.